subsys/bluetooth/audio/Kconfig.mediacontrol

```
@@ -0,0 +1,50 @@
+# Bluetooth Audio - Media control configuration options
+
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if BT_AUDIO
+
+config BT_MEDIACONTROL
+       bool "Support for media player and media player control"
+       help
+         Enables support for media player and for control of local and remote
+	 media players
+
+if BT_MEDIACONTROL
+
+config BT_MEDIACONTROL_LOCAL_PLAYER
+       bool "Support for local media player"
+       #TODO: Move MPL out of BT susbystem, change "select" to "depends"?
+       select BT_MPL
+       help
+         This option enables support for a local media player
+
+config BT_MEDIACONTROL_LOCAL_PLAYER_LOCAL_CONTROL
+       bool "Support for local control of local media player"
+       depends on BT_MEDIACONTROL_LOCAL_PLAYER
+       help
+         This option enables support for local application control of local
+	 media players
+
+config BT_MEDIACONTROL_LOCAL_PLAYER_REMOTE_CONTROL
+       bool "Support for remote control of local media player"
+       depends on BT_MEDIACONTROL_LOCAL_PLAYER
+       select BT_MCS
+       help
+         This option enables support for remote control of local media
+	 players, using GATT and the Media Control Service
+
+config BT_MEDIACONTROL_REMOTE_PLAYER
```

This is for a remote control, right?
Then perhaps 
```suggestion
config BT_MEDIA_REMOTE_CONTROL
```

---

In general I'd split the remote and local media player such that this doesn't rely on `BT_MEDIACONTROL`

---

Do you mean split the local player from the player control?

---

I mean splitting the local player control and the remote player control